### PR TITLE
OADP-3182: Split and Tidy up Release Notes

### PR DESCRIPTION
**Pull request by anarnold97**
### Jira

* [OADP-3182](https://issues.redhat.com/browse/OADP-3182)

### Version:

* OCP 4.12 → branch/enterprise-4.12
* OCP 4.13 → branch/enterprise-4.13
* OCP 4.14 → branch/enterprise-4.14
* OCP 4.15 → branch/enterprise-4.15

### Doc preview

* [OADP 1.3 release notes](https://68923--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/release-notes/oadp-release-notes)
* [OADP 1.2 release notes](https://68923--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/release-notes/oadp-release-notes-1-2)
* [OADP 1.1 release notes](https://68923--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/release-notes/oadp-release-notes-1-1)

### QE review:

* [ ] QE has approved this change.

### CODE REVIEW

**Review by PrasadJoshi12 — APPROVED** [review]

**Comment by PrasadJoshi12** [comment]
LGTM

**Comment by tmalove** [comment]
/remove-label peer-review-in-progress
/label peer-review-done

Don't forget to squash your commits.

**Review by adellape — COMMENTED** [review]
A few things to address before merge.

> @adellape - opinion required - do you think this is an improvement and makes more sense?

I think it is a worthwhile change, yes! Makes reading the latest version much cleaner for sure.

File: backup_and_restore/application_backup_and_restore/release-notes/oadp-release-notes-1-2.adoc [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+include::modules/oadp-release-notes-1-2-1.adoc[leveloffset=+1]
+
+include::modules/oadp-release-notes-1-2-0.adoc[leveloffset=+1]
+include::modules/oadp-backing-up-dpa-configuration-1-2-0.adoc[leveloffset=+3]
+include::modules/oadp-upgrading-oadp-operator-1-2-0.adoc[leveloffset=+3]
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../../backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc#migration-configuring-aws-s3_installing-oadp-aws[Configuring Amazon Web Services].
+* xref:../../../backup_and_restore/application_backup_and_restore/installing/oadp-using-data-mover-for-csi-snapshots-doc.adoc#oadp-using-data-mover-for-csi-snapshots-doc[Using Data Mover for CSI snapshots].
```
Comment (adellape) [inline backup_and_restore/application_backup_and_restore/release-notes/oadp-release-notes-1-2.adoc:25]: Not blocking merge for this, but just noticed periods on the ends of these bullets that could be removed in a later PR.

**Pull request merged**